notation.go

```diff
@@ -22,6 +24,31 @@ import (
 const annotationX509ChainThumbprint = "io.cncf.notary.x509chain.thumbprint#S256"
 
 var errDoneVerification = errors.New("done verification")
+var reservedAnnotationPrefixes = [...]string{"io.cncf.notary"}
+
+// RemoteSignOptions contains parameters for notation.Sign.
+type RemoteSignOptions struct {
+	// ArtifactReference sets the reference of the artifact that needs to be signed.
+	ArtifactReference string
+
+	// SignatureMediaType is the envelope type of the signature.
+	// Currently both `application/jose+json` and `application/cose` are
+	// supported.
+	SignatureMediaType string
+
+	// ExpiryDuration identifies the expiry duration of the resulted signature. Zero value
+	// represents no expiry duration.
+	ExpiryDuration time.Duration
+
+	// PluginConfig sets or overrides the plugin configuration.
+	PluginConfig map[string]string
+
```

Comment:
Can we embed SignOptions into RemoteSignOptions?

you can do is embed SignOptions inside RemoteSignOptions.
E.g.